composer/core/time.py

```diff
@@ -236,13 +236,15 @@ def _parse(self, other: object) -> Time:
                     To fix this warning, replace {other} with {other_parsed}."""))
             return other_parsed
 
-        raise NotImplementedError(f"Cannot convert type {other} to {self.__class__.__name__}")
+        raise TypeError(f"Cannot convert type {other} to {self.__class__.__name__}")
```

Comment:
This was a bug in the Timer; needed to fix since `Time == None` was erroring instead of returning False.